src/test/Microsoft.Extensions.DependencyModel.Tests/DependencyContextJsonReaderTest.cs

```diff
@@ -39,6 +38,27 @@ public void ReadsRuntimeTargetInfo()
             context.Target.RuntimeSignature.Should().Be("target-signature");
         }
 
+        [Fact]
+        public void ReadsRuntimeTargetInfoWithCommentsIsInvalid()
+        {
+            var exception = Assert.Throws<JsonReaderException>(() => Read(
```

Comment:
The tests target `netcoreapp2.0` so they don't run on non-netstandard2.0. We can add older TFMs to test the Newtonsoft code-paths.

Reply:
Based on offline discussion and https://github.com/dotnet/core-setup/issues/5034, it's not worth adding TFMs for this since we will remove the non-netstandard2.0 code paths.